emission/net/ext_service/habitica/proxy.py

```diff
@@ -19,7 +19,7 @@
 
 
 try:
-    key_file = open('conf/net/ext_service/habitica.json')
+    key_file = open('conf/net/ext_service/habitica.json.sample')
```

Comment:
revert. if this is not present, we will just disable habitica. It is worse to try and connect to a non-existent URL